pkgs/development/python-modules/arviz/default.nix

```diff
@@ -52,6 +55,7 @@ buildPythonPackage rec {
     pandas
     scipy
     setuptools
```

Comment:
Suspicious! There's no `build-system` argument and we're propagating `packagin` and `setuptools` without any comments as to why

Reply:
Yes. I think that this derivation was written this way is because all dependencies (even `setuptools`) are listed in [requirement.txt](https://github.com/arviz-devs/arviz/blob/v0.18.0/setup.py).

Reply:
I was able to move both `packaging` and `setuptools` to `build-system` without breaking the build.